integration_test/ops_agent_test.go

```diff
@@ -222,6 +222,13 @@ func installOpsAgent(ctx context.Context, logger *logging.DirectoryLogger, vm *g
 		}
 		return nil
 	}
+	// Removing flaky rhel-7 repositories due to b/265341502
+	if gce.IsRHEL7(vm.Platform) {
+		if _, err := gce.RunRemotely(ctx,
+			logger.ToMainLog(), vm, "", `sudo yum -y --disablerepo=rhui-rhel*-7-* install yum-utils && sudo yum-config-manager --disable "rhui-rhel*-7-*"`); err != nil {
+			return fmt.Errorf("yum check-update failed : %v", err)
```

Comment:
This error message doesn't match the actual yum commands. Since the error will already have the commands embedded in it, no need to repeat that, maybe just:

`return fmt.Errorf("disabling flaky repos failed : %w", err)`

Also, for new code prefer %w when wrapping an inner error with a new error message because it preserves the type of the underlying error, sorta. https://go.dev/blog/go1.13-errors

Reply:
Addressed in https://github.com/GoogleCloudPlatform/ops-agent/pull/1062/commits/585ed6fe32bd2c38005fd7f344bcae7993f4d980 .